specs/signing-and-verification-workflow.md

```diff
@@ -1,6 +1,6 @@
 # Signing and Verification Workflow
 
-This document describes how Notary v2 signs and verifies OCI artifacts.
+This document describes how Notary Project signs and verifies OCI artifacts.
```

Comment:
This document describes how **the** Notary Project signs and verifies OCI artifacts.